### DISCUSSION TITLE
dev server v2 not working on windows

### DISCUSSION THREAD

**Discussion by egcampbell** [remix-run/remix#6722] [Q&A] [answered]
I am using the remix server and using the scripts provided under How to use with MSW.  It works on mac (apple silicon) but when I run the command on windows, it can't find my files.  It says Unable to find package.json and gives me all of the v2 future flag warnings even though I have them all set to true in my config.  Any suggestions?

**Comment by mattf96s** [comment]
https://github.com/xHomu/remix-v2-server/tree/cjs-server.js#troubleshooting

**Reply by xHomu** [reply, marked as answer]
Specifically, on Windows you want to use the back slash to escape -c command:

`    "dev": "remix dev -c \"npm run dev:node\" --no-restart",`
